rotkehlchen/db/accounting_rules.py

```diff
@@ -204,6 +205,28 @@ def add_linked_setting(
                 f'Link of rule for {rule_property} to setting {setting_name} already exists',
             ) from e
 
+    def get_valid_entry_combinations(
+            self,
+            entries: list[HistoryBaseEntry] | list[tuple[int, HistoryBaseEntry]],
+    ) -> set[tuple[HistoryEventType, HistoryEventSubType, Optional[str]]]:
+        if len(entries) == 0:
+            return set()
+        combinations = {
+            (entry[1].event_type, entry[1].event_subtype, getattr(entry[1], 'counterparty', None))
+            if isinstance(entry, tuple)
+            else (entry.event_type, entry.event_subtype, getattr(entry, 'counterparty', None))
+            for entry in entries
+        }
```

Comment:
as I mentioned standarizing the format you can remove complexity from here

Reply:
Also you can make the transformation to NO_ACCOUNTING_COUNTERPARTY here

Reply:
so for this, I didn't transform to `NO_ACCOUNTING_COUNTERPARTY` here, it wasn't different from leaving as is, and would have been an additional import to this file, unless I didn't understand correctly